packages/essentials/utilities/timing/Timer.test.ts

```diff
@@ -0,0 +1,209 @@
+import { Timer, TimerEvent, TimerStatus } from "./Timer";
```

Comment:
I know it's out of the scope of this PR but I think we should reconsider our implementation and use CustomEvents https://developer.mozilla.org/en-US/docs/Web/API/CustomEvent @thijsdaniels